### ISSUE TITLE
[4.0] Articles Custom Field with position 'After Title' not rendered in category blog

### ISSUE THREAD

**Issue by Scrabble96** [joomla/joomla-cms#36468]
### Steps to reproduce the issue
In J4.0.5 create a new text field assigned to all categories.
In the Options tab, under 'Automatic Display', select 'After Title'.
Open an article that you will view as a single article and also as part of a category blog page

### Expected result
The field should be displayed after the Title (or after where the title would be if not shown)

### Actual result
It only appears in a single article view. 
It does not appear in the category blog display.

**Comment by brianteeman** [comment]
oops i didnt save the change to aftertitle

**Comment by Scrabble96** [comment]
This is what I see:

In a blog...
![1-custom-fields-blog-display](https://user-images.githubusercontent.com/36987494/147597953-f94dd149-9f34-467d-a849-144a1e93d09b.jpg)

And in the article view...
![2-custom-fields-article-display](https://user-images.githubusercontent.com/36987494/147597963-486655fc-b6a3-41a7-9004-4ed17e5fec2a.jpg)

**Comment by Scrabble96** [comment]
This is a fresh install of J4.0.5 today, using the Cassiopeia template.

**Comment by drmenzelit** [comment]
The field "after title" will be shown in the blog layout only when the intro text is set to "hide":
https://github.com/joomla/joomla-cms/blob/fcb74f6e3324340b173173f88e0febf0c45a0a1e/components/com_content/tmpl/category/blog_item.php#L58

![grafik](https://user-images.githubusercontent.com/9153168/147599394-fbb17b4e-2af1-43a7-81ec-21cbf2430a1a.png)

**Comment by brianteeman** [comment]
What is the reason for that @drmenzelit

**Comment by drmenzelit** [comment]
@brianteeman I have no idea... the code was introduced with this PR https://github.com/joomla/joomla-cms/pull/15497/files 5 years ago...

**Comment by brianteeman** [comment]
It is an easy change but I worry about b/c

**Comment by rdeutz** [comment]
I don't think we can make this change, because fields would be visibile in all blog views and I don't think that people expect this. I am not a fan of introducing new options but this seems to me the better way in this case. If it is really needed now an overwrite can do the job easily.

**Comment by brianteeman** [comment]
This should be closed for the reasons stated above

**Comment by MacJoom** [comment]
Thank you for your contribution - the issue was closed since no further input was given. Please write me a message if you think it is still relevant.
